Saule/Queries/Lambda.cs

```diff
@@ -50,15 +51,27 @@ internal static List<object> TryConvert(List<string> values, Type type)
             return parsedList;
         }
 
+        internal static object TryConvert(string value, Type type)
+        {
+            var converter = TypeDescriptor.GetConverter(type);
+            return converter.ConvertFromInvariantString(value);
+        }
+
         // Return value is used through reflection invocation
         // ReSharper disable once UnusedMethodReturnValue.Local
         private static Expression<Func<TClass, bool>> Convert<TProperty, TClass>(
             Expression<Func<TProperty, TProperty, bool>> expression,
-            TProperty constant,
+            List<TProperty> constant,
             MemberExpression propertyExpression,
             ParameterExpression parameter)
         {
-            var curriedBody = new FilterLambdaVisitor<TProperty>(propertyExpression, constant).Visit(expression.Body);
+            // initialize the expression with a always true one to make chaining possible
+            Expression curriedBody = Expression.Lambda(Expression.Constant(true));
```

Comment:
should be `false`? (`true || a == b || a == c || a == d` is always true)

Reply:
You are right my bad!